src/planetary_system.jl

```diff
@@ -136,6 +136,52 @@ function draw_power_law(n::Real, x0::Real, x1::Real, num_pl::Integer)
      ((x1^(n+1) - x0^(n+1)).*rand(num_pl) .+ x0^(n+1)).^(1/(n+1))
 end
 
+function draw_broken_power_law(n1::Real, n2::Real, x0::Real, x1::Real, xb::Real, num_pl::Integer)
+    #x0 and x1 are the lower and upper truncation limits, and xb is the break point, i.e. x0 <= xb <= x1 (all must be positive)
+    #n1 and n2 are the power law indices between x0 and xb, and xb and x1, respectively (can be positive or negative)
+
```

Comment:
What about adding a couple of asserts?
@assert x0 <= xb <= x1
@assert num_pl >= 1

Reply:
OK, this is a good idea.